data-prepper-core/src/main/java/org/opensearch/dataprepper/pipeline/router/RouterCopyRecordStrategy.java

```diff
@@ -134,4 +134,10 @@ public Collection<Record> getAllRecords(final Collection<Record> allRecords) {
         }
         return newRecords;
     }
+
+    // for testing
```

Comment:
Please make these Javadocs. They will "stick" with the function better this way.

```
/*
 * for testing
 */
```

Reply:
Also, make it package protected by removing `public`.

Reply:
It needs to be public because I am accessing it in a different package.